index.html

```diff
@@ -21,14 +24,186 @@
             <img class="bars" src="img/ic_menu.png" alt="">
         </nav>
     </div>
+
     <!-- toolbar section ends -->
 
+
+
     <!-- headline section starts -->
+
     <section class="header">
-        <h1 class="name">Lil-Snow</h1>
-        <p class="about">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don't hesitate and contact me.</p>
-        <button class="btn">Start collaboration</button>
+        <h1 class="name title_font">Lil-Snow</h1>
+        <p class="feature">Hello! I am a software developer! I can help you build a product, feature or website. Taka a
+            look of my works. If you like what you see and have a project you need coded, don't hesitate and contact me.
+        </p>
+        <button class="main_btn">Start collaboration</button>
     </section>
+
     <!-- headline section ends -->
+
+
+
+    <!-- work section starts -->
+    <section class="work">
+        <h1 class="title_font projectH1">Projects</h1>
+        <div class="showcase">
+
+            <!-- project 1 showcase -->
+            <div class="project">
+                <img class="project_img" src="img/Project1.png" alt="Project1">
+                <div class="menu">
+                    <h2 class="sub_title">Project name goes here</h2>
+                    <ul class="list">
+                        <li><button class="btn2">HTML/CSS</button></li>
+                        <li><button class="btn2">Ruby on Rails</button></li>
+                        <li><button class="btn2">JavaScript</button></li>
+                    </ul>
+                    <div class="btn3">
+                       <h2 class="text">See this project</h2>
+                       <img class="arrow" src="img/ic_arrow_right.png">
+                    </div>
+                </div>
+            </div>
```

Comment:
- [ ] Kindly use `section` tag to wrap the content of all the cards. It has more semantics that div. 👍 You can as well check the requirement for confirmation.